src/DFC_BusinessOntology.owl

```diff
@@ -9124,6 +10456,11 @@
         <IRI>#startDate</IRI>
         <Literal xml:lang="fr">date de début</Literal>
     </AnnotationAssertion>
+    <AnnotationAssertion>
+        <AnnotationProperty abbreviatedIRI="rdfs:comment"/>
+        <IRI>#stockLimitation</IRI>
+        <Literal xml:lang="en">Any limit on the stock of this particular listing. This ay differ from stock limits on the Product (for example if the Product is listed in multiple catalogues)</Literal>
```

Comment:
Should not we use "catalogs" instead of "catalogues"?

Reply:
Also should be "This **m**ay differ from stock limits..."